iocore/net/I_NetVConnection.h

```diff
@@ -644,6 +647,103 @@ class NetVConnection : public VConnection
   NetVConnection(const NetVConnection &) = delete;
   NetVConnection &operator=(const NetVConnection &) = delete;
 
+  enum ProxyProtocolVersion_t {
+    PROXY_VERSION_UNDEFINED,
+    PROXY_V1,
+    PROXY_V2,
+  };
+
+  enum ProxyProtocolData_t {
+    PROXY_PROTO_UNDEFINED,
+    PROXY_PROTO_SRC,
```

Comment:
Hmmm, this looked a little odd to me when reading the code. You are really treating this like a boolean in (almost?) all places where it's used? Why not just get rid of this, and have a boolean parameter where needed, like proxo_proto_is_src or some such.

I don't care particularly strongly on it though.

Reply:
I am converting both of these enums to enum classes, and reducing the names to UNDEFINED, VI, etc.  As for the ProxyProtocolData enum, I use this as a sort of switch for the get_ and set_proxy_protocol_addr(). E.g. the intent is to call set_proxy_protocol_src_addr() which sets the SRC or DST flag before calling set_proxy_protocol_src_addr().  If this doesn't work for you guys, I can change it to a simple bool.  I was just looking for something I could expand on down the road.